### PR TITLE
Allow wal-g to use gcs

### PR DESCRIPTION
**Pull request by pitabwire**
As discussed in issue https://github.com/zalando/spilo/issues/319 @CyberDem0n please see this

### CODE REVIEW

File: postgres-appliance/Dockerfile [hunk 1]
```diff
@@ -367,7 +367,7 @@ RUN export DEBIAN_FRONTEND=noninteractive \
 # Install patroni, wal-e and wal-g
 ENV PATRONIVERSION=1.5.5
 ENV WALE_VERSION=1.1.0
-ENV WALG_VERSION=v0.2.4
+ENV WALG_VERSION=v0.2.6
```
Comment (CyberDem0n) [inline postgres-appliance/Dockerfile]: This is not the latest version.

Comment (pitabwire) [reply postgres-appliance/Dockerfile]: This is updated upropriately

**Comment by avaczi** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Comment by CyberDem0n** [comment]
Merged, thank you!

**Pull request merged**